test/jobs/payout/uphold_job_test.rb

```diff
@@ -23,7 +23,7 @@ class UpholdJobTest < ActiveSupport::TestCase
     uphold_job = Payout::UpholdJob.new
     uphold_job.perform({ payout_report_id: payout_reports(:one).id }.to_json)
     visited_publisher_id = nil
-    assert IncludePublisherInPayoutReportJob.jobs.count > 1
```

Comment:
Why did this change?

Reply:
It's technically fine at `> 1` but I think it should have been written as `>= 1` or `> 0`